orb/commands/@drupal.yml

```diff
@@ -65,10 +65,15 @@ drupal-composer-install:
         key: <<parameters.cache-version>>-dependencies-{{ checksum "composer.lock" }}-<<parameters.install-dev-dependencies>>
 
 drupal-docker-build:
+  parameters:
+    build_context:
```

Comment:
This parameter probably needs to be renamed. We have more than one build context, how about calling this one `nginx_build_context`?